draft-ietf-rats-msg-wrap.md

```diff
@@ -319,6 +319,26 @@ with the following wire representation:
    03                                 # unsigned(3)
 ~~~
 
+# CMW Collections {#cmw-coll}
+
+Composite and layered attesters produce evidence made of multiple evidence "bits".
+
+For example, in data centre servers, it is not uncommon for separate attesting environments (AE) to serve a subsection of the entire machine.
+One AE measures and attests to what was booted on the main CPU, another AE measures and attests to what was booted on a SmartNIC plugged into a PCIe slot, and a third one measures and attests to what was booted on the machine's GPU.
+
+To address the composite attester use case, this document defines a CMW "collection" as a container that aggregates several CMW evidence "bits," each with its unique name.
+
+The CMW collection ({{fig-cddl-collection}}) is defined as a CBOR map or JSON object with CMW values.
+The keys can be strings or integers, representing a unique label that identifies an evidence "bit" within the composite evidence.
+
+~~~ cddl
+{::include cddl/cmw-collection.cddl}
+~~~
+{: #fig-cddl-collection artwork-align="left"
+   title="CDDL definition of the CMW collection format"}
+
+Although initially designed for the composite attester use case, the CMW collection can be repurposed for other use cases requiring CMW aggregation.
```

Comment:
```suggestion
Although initially designed for the composite attester use case,
the CMW collection can be repurposed for other use cases such as CMW aggregation.
CMW aggregation might be used by a lead Attester to bundle Evidence obtained from multiple
Attesters that rely on a lead Attester for connectivity to a remote Verifier.
```

Reply:
hmm, L342-343 seem to be restating the composite attester use case, which is already explained in L324-329.

Reply:
Maybe try rewording L324-329 to be more succinct and less colloquial?